### PR TITLE
add account list contents, style and functionality, fix running totals

### PR DESCRIPTION
**Pull request by nader-fares**

### CODE REVIEW

File: src/screens/Overview.tsx [hunk 1]
```diff
@@ -1,46 +1,112 @@
...
                 <View style={styles.label}>
-                    <Text style={styles.text}>Accounts</Text>
+                    <Text style={styles.titleText}>Accounts</Text>
                 </View>
-                    <View style={{position: 'absolute', bottom: 150, width: 333, height: 40, display: 'flex', flexDirection: 'row', justifyContent: 'center', alignItems: 'center', padding: 0 }}>
+
+                
+
+                <View style={{top: 150, width: '100%', paddingHorizontal: 10}}>
+                    {accounts.map(account => (
```
Comment (conroyr41) [inline src/screens/Overview.tsx:97]:
```suggestion
                    {accounts.map((account, index) => (
```

File: src/screens/Overview.tsx [hunk 2]
```diff
@@ -1,46 +1,112 @@
...
+                        <View style={{ width: '100%', display: 'flex', justifyContent: 'space-between', flexDirection: 'row', padding: 10}}>
```
Comment (conroyr41) [inline src/screens/Overview.tsx:98]:
```suggestion
                        <View key = {index} style={{ width: '100%', display: 'flex', justifyContent: 'space-between', flexDirection: 'row', padding: 10}}>
```
```suggestion
                        <View key = {index} style={{ width: '100%', display: 'flex', justifyContent: 'space-between', flexDirection: 'row', padding: 10}}>
```

Comment (conroyr41) [inline src/screens/Overview.tsx:98]:
```suggestion
                        <View key = {index} style={{ width: '100%', display: 'flex', justifyContent: 'space-between', flexDirection: 'row', padding: 10}}>
```

**Review by conroy-ricketts — CHANGES_REQUESTED** [review]
This will fix the error in your code.

**Pull request merged**